R/piping-model.R

```diff
@@ -88,39 +84,59 @@ model.rxModelVars <- model.rxode2
       }
     })
   }
-  return(rxUiCompress(rxui$fun()))
+  rxUiCompress(rxui$fun())
+}
+
+# Determine if the input is an endpoint by being 3 long and the call part being
+# a tilde
+.isEndpoint <- function(expr) {
+  matchesLangTemplate(expr, str2lang(". ~ ."))
+}
+
+# Determine if the input is an assignment by being 3 long and the call part
+# being either the left arrow, right arrow, or equal sign
+.isAssignment <- function(expr) {
+  matchesLangTemplate(expr, str2lang(". <- .")) |
+    matchesLangTemplate(expr, str2lang(". = .")) |
+    matchesLangTemplate(expr, str2lang(". -> ."))
```

Comment:
`". -> "` is not supported.  

Also use logical `or` (that is `||`).  This  should never be a vector and vectorized expressions should throw an error.

Reply:
I was thinking that it could eventually be vectorized, but it is not near there yet.  So, I've made it use `||`.

I removed the right arrow, too.

Reply:
Thanks.  

I think you forgot to push.